vowpalwabbit/cost_sensitive.h

```diff
@@ -17,6 +17,13 @@ struct wclass
   float partial_prediction;  // a partial prediction: new!
   float wap_value;  // used for wap to store values derived from costs
   bool operator==(wclass j) {return class_index == j.class_index;}
+
```

Comment:
This is a bit of a mess since it's contaminating a label data structure with a bunch of nonlabel stuff.

Reply:
I can move most of the stuff to cs_active but query_needed is set in cs_active but used in search. I'm not sure of a better place to have it?